spaceship/lib/spaceship/test_flight/tester.rb

```diff
@@ -20,11 +20,13 @@ def self.all(app_id: nil)
       client.testers_for_app(app_id: app_id).map { |data| self.new(data) }
     end
 
-    # @return (Spaceship::TestFlight::Tester) Returns the tester matching the parameter
-    #   as either the Tester id or email
-    # @param email (String) (required): Value used to filter the tester, case insensitive
-    def self.find(app_id: nil, email: nil)
```

Comment:
Do we want to deprecate this method name and point it at the new method to keep this from breaking other `spaceship` users?